biztalk/core/logic-app-adapter.md

```diff
@@ -47,6 +47,9 @@ There are a few steps involved for BizTalk Server to receive messages from a log
 
 ### Step 1: Install the Logic App adapter
 
+> [!NOTE]
+> Starting with **BizTalk Server 2020**, the logic app adapter is included with the BizTalk Server installation. If you are using BizTalk Server 2020, please skip this step and proceed to Step 2.
```

Comment:
We could also say this step is required only if you are using BizTalk 2016. That makes it more future ready - just a thought.

Reply:
Updated as per offline discussion